### ISSUE TITLE
doubts about requires_grad = False

### ISSUE THREAD

**Issue by 1160300929** [pytorch/pytorch#25471]
what happens when requires_grad is false?

**Comment by SsnL** [comment]
Ops on this tensor are not tracked for gradient computation. Use discuss.pytorch.org for questions please.
